### ISSUE TITLE
Reason why demos license was "changed" from Apache 2.0 to MIT? Approved by all?

### ISSUE THREAD

**Issue by sandraros** [abap2xlsx/demos#52]
@gregorwolf @larshp I just saw that the [demos](https://github.com/abap2xlsx/demos) repository was [created](https://github.com/abap2xlsx/abap2xlsx/issues/861) under the **MIT license**, although the [abap2xlsx](https://github.com/abap2xlsx/abap2xlsx) repository is under **Apache 2.0**. It's weird because the demo programs were initially created in the abap2xlsx repository under the Apache 2.0 license, so I'm expecting that the demos repository should be under the same license.

I have nothing against changing the license, but is it an error or did you [both agree](https://opensource.stackexchange.com/questions/9077/relicensing-from-apache-2-0-to-apache-or-mit)? (and same question for any other owner I'm not aware of).

demos:

![image](https://github.com/abap2xlsx/demos/assets/34005250/0d1056dd-5180-4f5d-8e1d-b025bd18508b)

abap2xlsx:

![image](https://github.com/abap2xlsx/demos/assets/34005250/ee5c2f81-f291-4eca-b7a5-bf2047590870)

**Comment by gregorwolf** [comment]
I think this was a mistake when the repository was created.

**Comment by larshp** [comment]
yea, probably just change it to Apache 2

tho, I dont think its a problem for anyone

**Comment by sandraros** [comment]
Thanks, done by #56.
I have set the demos copyright to "Copyright 2022 abap2xlsx Contributors" (previously Copyright (c) 2022 abap2xlsx). Not sure about the year, maybe should be 2010, the abap2xlsx creation date, but I guess here it's not important.

I also fixed the main abap2xlsx copyright ([abap2xlsx license: unchanged original "Copyright {yyyy} {name of copyright owner}](https://github.com/abap2xlsx/abap2xlsx/issues/1153)), changed to "Copyright 2010 abap2xlsx Contributors" which is the year abap2xlsx was created.
